content/master/cli/command-reference.md

```diff
@@ -18,10 +18,14 @@ The following flags are available for all commands.
 | Short flag | Long flag   | Description                  |
 |------------|-------------|------------------------------|
 | `-h`       | `--help`    | Show context sensitive help. |
-| `-v`       | `--version` | Print version and exit.      |
 |            | `--verbose` | Print verbose output.        |
 {{< /table >}}
 
+## version
+
+The `crossplane version` command returns the version of the Crossplane CLI you
```

Comment:
The `version` command now also returns the version of the Crossplane software running in the control plane too, which could be an entirely different version than that of the CLI. Can you update to make that clear here?